caddytest/integration/caddyfile_adapt/reverse_proxy_load_balance_wrr.txt

```diff
@@ -0,0 +1,71 @@
+:8884
+
+reverse_proxy 127.0.0.1:65535 127.0.0.1:35535 {
+	lb_policy weighted_round_robin "10, 1"
```

Comment:
I find this syntax kinda strange and not idiomatic - typically arguments in the Caddyfile are space separated, not comma separated. You can use `d.RemainingArgs()` to get a slice of tokens from the same-line.

Reply:
Done.